api/csi.go

```diff
@@ -150,3 +152,81 @@ type CSIVolumeDeregisterRequest struct {
 	VolumeIDs []string
 	WriteRequest
 }
+
+// CSI Plugins are jobs with plugin specific data
+type CSIPlugins struct {
+	client *Client
+}
+
+type CSIPlugin struct {
+	ID        string
+	Driver    string
+	Type      CSIPluginType
+	Namespace string // FIXME all jobs in the same namespace?
+	Jobs      map[string]map[string]*Job
```

Comment:
Could you document what this mapping refers to?

Reply:
Yes! that's done